lib/Service/Gateway/SMS/Provider/AndroidGSMmodem.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\TwoFactorGateway\Service\Gateway\SMS\Provider;
+
+use Exception;
+use OCA\TwoFactorGateway\Exception\SmsTransmissionException;
+use OCP\Http\Client\IClient;
+use OCP\Http\Client\IClientService;
+
+class AndroidGSMmodem implements IProvider {
```

Comment:
```suggestion
class AndroidGsmModem implements IProvider {
```

https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

Also please update all references and the file name